src/index.html

```diff
@@ -7,8 +7,32 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.cdnfonts.com/css/roboto?styles=14400" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="#logo">
+          <img class="img" src="./images/logo.png"
+          alt="img">
+          </img>
+        </a>
+        <nav class="nav class=">
+          <ul class="nav__list">
+            <li class="header_text"><a href="#apple" class="is-active">Apple</a></li>
+            <li class="header_text"><a href="#samsung" class="is-active">Samsung</a></li>
+            <li class="header_text"><a href="#smartphones" class="is-active">Smartphones</a></li>
+            <li class="header_text">
+              <a href="#leptops&computer" class="is-active"
+              data-qa="hover">Laptops & Computers</a>
+            </li>
```

Comment:
```suggestion
            <li class="header_text">
              <a 
                href="#leptops&computer" 
                class="is-active"
                data-qa="hover"
              >
                Laptops & Computers
               </a>
            </li>
```
when tag has more two attributes - each should be on a new line